app/views/_includes/forms/training-details.html

```diff
@@ -72,7 +72,7 @@ <h1 class="govuk-heading-l">{{pageHeading}}</h1>
     classes: "govuk-label--s"
   },
   hint: {
-    text: "Your internal identifier for this trainee or candidate"
+    text: "You can assign your own ID for this trainee if it'll help you find them in your records."
```

Comment:
```suggestion
    text: "You can assign your own ID for this trainee if it’ll help you find them later"
```

I feel it's more about them finding their trainees in *our* system than in their records.